frontend/src/App.tsx

```diff
@@ -24,10 +19,14 @@ const App = () => {
             <ThemeProvider theme={THEME_DARK}>
                 <BrowserRouter>
                     <Routes>
-                        <Route element={<Outlet />}>
+                        <Route>
                             <Route
```

Comment:
Hierarchy of routers should be like:

/Something
/Some
/ *Empty*

Cause if u type something in url, then here it will be matched letter be letter.

In the case above, check S, o, m, e (And here if u have /Some above /Something, you will never be able to hit /Something. 

Rerange Routers